scene/2d/physics/collision_shape_2d.cpp

```diff
@@ -208,6 +208,12 @@ bool CollisionShape2D::is_disabled() const {
 	return disabled;
 }
 
+bool CollisionShape2D::contains_point(const Vector2 &point)
+{
+  return shape->contains_point((point - collision_object->shape_owner_get_transform(owner_id).get_origin()).
```

Comment:
Could the Transform2D be directly applied instead? It looks to me like this does not properly take scale into account.

Reply:
It doesn't take scale into account currently, that wasn't so much intentional so much as I wasn't planning on using scale for my own project at the time I made the pull request. Adding scaling functionality should be trivial though, so I'll probably add that in when I get the chance